app/controllers/admin/events_controller.rb

```diff
@@ -58,6 +59,10 @@ def set_event
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def event_params
-      params.require(:event).permit(:title, :subtitle, :description, :year, :logo)
+      params.require(:event).permit(:title, :subtitle, :overview, :year, :logo,
+        event_info_attributes: [:id, :synopsis, :description, :video_link])
+        .tap do|event_params|
+          event_params.require(:event_info_attributes)
```

Comment:
why do you need this line?

Reply:
Did you mean the "event_params.require(:event_info_attributes)"? it is to ensure that :event_info_attributes is also sent whenever user updated or created an event.